src/lib/i18n.ts

```diff
@@ -1,47 +1,98 @@
 import i18n from "i18next";
 import Backend from "i18next-http-backend";
+import { useMemo } from "react";
 import {
-	type UseTranslationResponse,
 	initReactI18next,
 	useTranslation,
 } from "react-i18next";
+import { z } from "zod";
+import { zodI18nMap } from "zod-i18n-map";
+import {
+	default as ZodEnUs,
+	default as ZodNlNl,
+} from "zod-i18n-map/locales/nl/zod.json";
+
```

Comment:
add so it can be reused:
```
const FALLBACK_LNG = "nl-NL";
```

Reply:
We may even put the user defined language in LS and take it from there. So when choosing a different language in the UI and refreshing, you keep your choice.